fr.handbrake.ghb.Plugin.dvdcss.json

```diff
@@ -0,0 +1,31 @@
+{
+    "app-id": "fr.handbrake.ghb.Plugin.dvdcss",
+    "branch": "1",
```

Comment:
do you need a branch? Unless there are api/abi breaks involved there is no need to have a branch.